edward/inferences.py

```diff
@@ -208,6 +214,10 @@ def initialize(self, n_iter=1000, n_data=None, n_print=100,
         init = tf.initialize_all_variables()
         init.run()
 
+        # Start input enqueue threads.
```

Comment:
is it possible to make this optional or have some stat for whether there are any queue runners? otherwise if the user escapes (ctrl+c) we'll have to catch that and stop the threads, e.g.
```
# Start input enqueue threads.
coord = tf.train.Coordinator()
threads = tf.train.start_queue_runners(sess=sess, coord=coord)
# training ops
...
# if training interrupted, or escape sequence
coord.request_stop()
coord.join(threads)
sess.close()
```